docker/docker-compose.yaml

```
@@ -0,0 +1,11 @@
+version: "3.8"
+
+services:
+  cln:
+    build:
+      context: ../
+      dockerfile: ./docker/Dockerfile.integration_test
+    volumes:
+      - ./client/:/workdir
+      - ./comm/:/workdir
+      - ./plugin/:/workdir
```

```suggestion
    volumes:
      - ./client/:/workdir
      - ./comm/:/workdir
      - ./plugin/:/workdir
```

Mh! yes this can be good if you want to test in the local machine too, but for now lets keep simple